shared.sh

```diff
@@ -2,10 +2,13 @@
 # Shared definitions for buildroot scripts
 
 # The defconfig from the buildroot directory we use for qemu builds
-QEMU_DEFCONFIG=configs/qemu_aarch64_virt_defconfig
+
```

Comment:
@steveantony5 I'd like to keep the QEMU defconfig lines to make it easy to switch back and forth.  See example beaglebone implementation at https://github.com/cu-ecen-5013/buildroot-assignments-base/pull/1/files#diff-1a2ebd708dfb11aebd676f9d9f26e69dR6

Reply:
@dwalkes  Added as said